### PR TITLE
🏗  Use new key for storage

### PR DESCRIPTION
**Pull request by estherkim**
Once merged, I'll delete GCP_TOKEN from CircleCI settings

### CODE REVIEW

File: .circleci/setup_storage.sh [hunk 1]
```diff
@@ -35,11 +35,11 @@ echo $(GREEN "Setting up Cloud SDK environment...")
 (set -x && echo "source ~/google-cloud-sdk/path.bash.inc" >> $BASH_ENV)
 source $BASH_ENV
 
-echo $(GREEN "Extracting credentials...")
-openssl aes-256-cbc -k $GCP_TOKEN -in ./build-system/common/sa-travis-key.json.enc -out sa-travis-key.json -d
```
Comment (rsimha) [inline .circleci/setup_storage.sh:39]: Happy to see this go away!

File: .circleci/setup_storage.sh [hunk 2]
```diff
@@ -35,11 +35,11 @@ echo $(GREEN "Setting up Cloud SDK environment...")
...
+echo $(GREEN "Writing key to json file...")
+echo $STORAGE_TOKEN > storage-key.json
 
 echo $(GREEN "Authenticating with GCP storage...")
-gcloud auth activate-service-account --key-file=sa-travis-key.json
+gcloud auth activate-service-account --key-file=storage-key.json
```
Comment (rsimha) [inline .circleci/setup_storage.sh:42]: Yay, logs are less noisy than before: https://app.circleci.com/pipelines/github/ampproject/amphtml/2973/workflows/ef569347-0472-48d6-aba3-86fddf2a168e/jobs/34847/parallel-runs/0/steps/0-104

Any idea how to get rid of the access warning and the survey messages?

**Review by rsimha — APPROVED** [review]
👍

File: .circleci/setup_storage.sh (hunk 2 above)
Comment (estherkim) [reply .circleci/setup_storage.sh:40]: Got rid of a few, not the survey though.

**Pull request merged**